### PR TITLE
PRM Automated Test - LPS-188520 | master

### PR DESCRIPTION
**Pull request by marinaasant0s**
[LPS-188520](https://liferay.atlassian.net/browse/LPS-188520) -  Automate PRMMDFRequestActivities#CanSubmitMDFWithDigitalMarketingEmailCampaignActivity

### CODE REVIEW

**Comment by marinaasant0s** [comment]
ci:test:sf

**Comment by marinaasant0s** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro [hunk 1]
```diff
@@ -681,6 +683,56 @@ definition {
...
+					key_fieldName = "activities[${activityIndex}].activityDescription.nurtureDripCampaign",
+					locator1 = "PRMMDFRequest#INPUT_BUTTON");
+			}
+
+			if (${nurtureDripCampaign} == "Yes") {
+				Type(
+					key_fieldName = "activities[${activityIndex}].activityDescription.manySeries",
+					locator1 = "PRMMDFRequest#TEXT_FIELD",
+					value1 = "Many Series");
+			}
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro:710]: Hi @marinaasant0s, this condition should be inside the previous condition.
The logic should be something like:
`If nurtureDripCampaign` is set, then select the input button as `yes` or `no`.
   ` If input button == yes,` then `fill the field`.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro [hunk 2]
```diff
@@ -681,6 +683,56 @@ definition {
...
+					key_fieldName = "activities[${activityIndex}].activityDescription.assetsLiferayRequired",
+					locator1 = "PRMMDFRequest#INPUT_BUTTON");
+			}
+
+			if (${requireLiferay} == "Yes") {
+				Type(
+					key_fieldName = "activities[${activityIndex}].activityDescription.assetsLiferayDescription",
+					locator1 = "PRMMDFRequest#TEXT_FIELD",
+					value1 = "Require Liferay");
+			}
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro:724]: Hi @marinaasant0s, apply the same logic here, please!

**Review by AfonsoHSantos — CHANGES_REQUESTED** [review]

**Comment by marinaasant0s** [comment]
ci:test:sf

**Review by AfonsoHSantos — APPROVED** [review]
Hi @marinaasant0s, thanks for working on it.
The code looks good to me!
nice

**Comment by AfonsoHSantos** [comment]
ci:forward

**Pull request closed without merging**